src/api/users/user.controller.ts

```diff
@@ -21,9 +22,10 @@ export class UserContoller{
     @ApiResponse({ status: 400, description: 'Validation failed'})
     @ApiResponse({ status: 422, description: 'Email are taken'})
     @UsePipes(new ValidationPipe())
-    async createUser(@Body() createUserDto: CreateUserDto): Promise<UserResponseInterface> { 
+    async createUser(@Body() createUserDto: CreateUserDto, @Res() response: Response): Promise<Response> { 
         const user = await this.userService.createUser(createUserDto);
-        return this.userService.buildUserResponse(user);
+        response.set({ 'authorization':  this.userService.generateJwt(user)});
+        return response.status(201).json({ message: "User create" });
```

Comment:
Can we return `user` defails here? It will be useful for FE as it won't need to do another request to fetch user data.

Reply:
- RESOLVED -